tests/ext/start_span_with_all_properties.phpt

```diff
@@ -63,7 +65,7 @@ array(2) {
     ["start"]=>
     int(%d)
     ["duration"]=>
-    int(2000000%d)
+    int(200000%d)
```

Comment:
Why are we losing a 0 here?

Reply:
test is flaky with too much precision :-/